src/main/java/seedu/address/logic/commands/KeyCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+//@@author lohtianwei
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+import com.google.common.hash.Hashing;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Sets lock in model to true/false depending on current state
+ */
+public class KeyCommand extends Command {
+    public static final String COMMAND_WORD = "key";
+    public static final String COMMAND_ALIAS = "k";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Toggles the lock on MTM. "
+            + "Functionality of MTM reduced\n"
```

Comment:
I think would be better to better phrase this such that non-technical users can understand 

> Limits the functionality of MTM when it is locked.
> In the locked state only <list of commands> can work.